src/utils/isOnlyEmoji.ts

```diff
@@ -1,8 +1,35 @@
 const emojiRegex =
-  /(<a?(:\w+:\d+)>|\p{Emoji_Presentation}|\p{Extended_Pictographic}|\p{Emoji_Component})\s*/gu
+  /(<a?(:\w+:\d+)>|\p{Emoji_Presentation}|\p{Extended_Pictographic}|\p{Emoji_Component}|:\w+:\s*)/gu
 
 export default (msg: string): boolean => {
   const emoji = msg.match(emojiRegex)
+  //the issue of #99(https://github.com/creatorsgarten/kaogeek-discord-bot/pull/99) only happend with only number message,
+  //so detect number in message and return false if there is.
+  if (emoji !== null) {
+    const unicoded = emoji.map((emo) => {
+      return emo.codePointAt(0)?.toString(16)
+    })
+    for (const i of unicoded) {
+      if (i !== undefined && isNumber(i)) {
```

Comment:
I use include with string array to avoid typecasting from string to int.

Reply:
I think both methods can avoid typecasting and achieve the same result in different ways.
anyway if you concerned about the case of i='301' yeah my statement will return true, which is not the same as your function.
But because of regex 
`/(<a?(:\w+:\d+)>|\p{Emoji_Presentation}|\p{Extended_Pictographic}|\p{Emoji_Component}|:\w+:\s*)/gu`
we can make sure that `const emoji = msg.match(emojiRegex)` will contain only emoji or number
In example when unicode ID: 779(which is hex string i='301') send to chat it not detect as emoji by regex work

If I have missed anything, please let me know krubb

Reply:
I have reread again, and I have to apologize about my wrong understanding of string comparison.  Yes, your method also avoids typecasting, but we cannot prove that there is and will always be no emoji in form of '3x...' string, if there is or will be in future. the logic will break.

Reply:
we can prove it by [Message Formatting](https://discord.com/developers/docs/reference#image-formatting:~:text=Standard%20Emoji,a%3Ab1nzy%3A392938283556143104%3E)

Reply:
it will always be fine for custom discord emoji as long as they do not change, but some of Unicode are still available and we don't know which will be filled into those empty slot.